pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "marshmallow-recipe"
+dynamic = ["version"]
+description = "Bake marshmallow schemas based on dataclasses"
+readme = {file = "README.md", content-type = "text/markdown"}
+requires-python = ">=3.12"
+license = {text = "MIT"}
+authors = [
+    {name = "Yury Pliner", email = "yury.pliner@gmail.com"}
+]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
+    "Operating System :: OS Independent",
+    "Development Status :: 5 - Production/Stable",
+]
+dependencies = [
+    "marshmallow>=2.20.5,<4",
+]
+
+[project.urls]
+Homepage = "https://github.com/Pliner/marshmallow-recipe"
```

Comment:
Should this be [https://github.com/anna-money/marshmallow-recipe](https://github.com/anna-money/marshmallow-recipe)?